community-operators/bundles/cluster-logging/bundle.0.0.1.yaml

```diff
@@ -0,0 +1,719 @@
+---
+# Source: operator-artifacts/templates/cluster-logging.yaml
+
```

Comment:
I did not see the comments in the previous bundle I reviewed. It it intentional? If we are using a tool to generate  bundle(s), shouldn't they all have the same format?

Reply:
@awgreene Abu brings up a good point. The dashes and comment are left by helm during bundle generation. Since the script leaves these comments it's better to leave it and not intervene manually to reduce potential errors. Thoughts?

Reply:
For now let's leave the files generated by the script alone - once the operator-courier is implemented this won't be an issue.